### PR TITLE
Added theme related code and images.

### PR DESCRIPTION
**Pull request by mark-dce**

### CODE REVIEW

File: app/assets/stylesheets/wustl.scss [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+    margin: 0;
+}
+div#site-search form {
+    margin: 0;
+    padding: 0;
+}
+nav#site-actions {
+    padding-top: 0;
+}
+footer#footer {
```
Comment (jcoyne) [inline app/assets/stylesheets/wustl.scss]: If you use scss here you'd write it like this:

``` scss
footer#footer {
    margin: 30px 0 0 0;
    width: auto;
    text-align: left;
    ul { 
       margin: 0;
       padding: 0;
       li {
         list-style-type: none;
       }
    }
}
```

File: app/views/shared/_title_bar.html.erb [hunk 2]
```diff
@@ -1,13 +1,13 @@
         <div class="container">
           <div id="title-bar" class="row">
-            <div id="site-search" class="col-md-6">
+            <div id="site-search" class="col-md-10">
               <%= render 'shared/site_search' if show_site_search? %>
             </div>
-
+	    <!--
```
Comment (jcoyne) [inline app/views/shared/_title_bar.html.erb]: There's no reason to comment this, may as well just remove it. We can always get it back from the git history.

File: config/locales/curation_concerns.en.yml [hunk 3]
```diff
@@ -0,0 +1,117 @@
+en:
+  curation_concerns:
```
Comment (jcoyne) [inline config/locales/curation_concerns.en.yml]: You should not copy this whole file here.  You only need to put the keys you want to change in your `config/locales/en.yml` if you copy the whole file, upgrading curation_concerns will be more difficult.

**Comment by mark-dce** [comment]
@informaticianme - we're going to close this PR for now while we finish getting the delete issue resolved.  Once we have a final release candidate, we can issue a new PR.

**Pull request closed without merging**